docs/connector/08-02-maintain-secure-connection-with-compass.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Maintain a secure connection with Compass
+type: Tutorials
+---
+
+After you have established a secure connection with Compass, you can fetch the configuration details and renew the client certificate before it expires.  
+To renew the client certificate, follow the steps in this tutorial.
+
+## Prerequisites
+
+- [OpenSSL toolkit](https://www.openssl.org/docs/man1.0.2/apps/openssl.html) to create a Certificate Signing Request (CSR), keys, and certificates which meet high security standards
+- [Established secure connection with Compass](08-01-establish-secure-connection-with-compass.md)
```

Comment:
Maybe this prerequisite should be the last one on the list, as the user must have Compass, Application, and Runtime first.